bgc_data/nrs_derived_indices_data.sql

```diff
@@ -217,7 +217,7 @@ pigments_avg AS (
 SELECT m."Project",
        m."StationName",
        m."TripCode",
-       m."SampleTime_local"::date AS "SampleDateLocal",
+       m."SampleTime_Local"::date AS "SampleDateLocal",
```

Comment:
While we're making these more consistent, this should be "SampleDate_Local" as in all the other products (probably needs to be updated in the product spec too)

Reply:
I'm not sure I understand. Maybe you meant that this variable should be left as `SampleTime_local` as in the other products (i.e., chemistry, tss, picoplankton and pigments) to be consistent? Although, I have a feeling that, the conversion to date for local time, it was a specific request by Claire for the derived indices. 
@clairedavies how would you like the derived indices time variables to be shown in the final product? Note that chemistry, tss, picoplankton and pigments have `SampleTime_Local` and `SampleTime_UTC`
Thus, in derived indices could be shown as: 
`SampleDateLocal` and `SampleTime_UTC` (i.e., no hh:mm for local time) or
`SampleTime_Local` and `SampleTime_UTC` (i.e., consistend with other poducts)

Reply:
Marty may have been referring to the lack of _ , ie. SampleDateLocal vs SampleDate_Local.
Looking at the current product on geoserver there is a UTC timestamp and a Local date. Please can we use the local time stamp from the bgc_trip data for this and have SampleTime_Local and SampleTime_UTC. It is mostly about plankton and so that time is applicable.
The only product that currently doesn't have a timestamp is the combined BGC data as the times are different for the array of samples we show. Although I have had some feedback that a time is required here. Although the samples are taken at different times it seems an indicative time would be useful. Seeing as we are making these changes it would be good if you could please use the earliest timestamp from the chemistry data for each record in this table. So then all the products would be Time and not Date. Let me know if that needs more explanation.

Reply:
> Looking at the current product on geoserver there is a UTC timestamp and a Local date. Please can we use the local time stamp from the bgc_trip data for this and have SampleTime_Local and SampleTime_UTC. It is mostly about plankton and so that time is applicable.

Fixed in commit b21b38ddd854dabfaa99a0bd4eae2c5d3d0a5e84